_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,132 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
```

Comment:
With the current applicability, `<td>` would not be applicable but `<td role="row">` would. This should most likely be added:

> [...], except if the element has an implicit semantic role that is identical to the explicit semantic role.